pkg/controller/migmigration/migmigration_controller.go

```diff
@@ -143,10 +146,12 @@ func (r *ReconcileMigMigration) Reconcile(request reconcile.Request) (reconcile.
 	}
 
 	// Set values.
-	log.SetValues("migration", migration.Name)
+	log.SetValues("migration", request)
```

Comment:
question: should this be migmigration?

Reply:
I think we don't have the `mig` prefix in other controllers (`cluster`, `analytic`, `plan`), so kept it consistent.